it/v1/endpoints/RetrieveChargeHistoryControllerISpec.scala

```diff
@@ -74,6 +74,39 @@ class RetrieveChargeHistoryControllerISpec extends IntegrationBaseSpec {
           response.json shouldBe mtdResponse
           response.header("Content-Type") shouldBe Some("application/json")
         }
+
+      }
+      "return a 500 status code" when {
+        "des returns multiple errors" in new Test {
```

Comment:
all good, can you change the description something like this **des returns errors that map to DownstreamError**

Reply:
but this test is specifically for the scenario in which multiple errors are returned from DES

Reply:
never mind, I thought this was on the other service. Ignore me.

Reply:
Done